projects/plugins/jetpack/extensions/blocks/tiled-gallery/styles.native.scss

```diff
@@ -0,0 +1,28 @@
+.galleryAppender {
+	padding-top: $grid-unit-20;
+}
+
+.horizontalBorder {
+	border-top-width: $border-width;
+	border-color: $light-gray-400;
+}
+
+.horizontalBorderDark {
+	border-color: $gray-70;
+}
+
+.is-style-squared {
+	aspect-ratio: 1;
+}
+
+.is-style-rectangular {
+	aspect-ratio: 1;
+}
+
+.cellRowStyles {
+	flex-shrink: 1;
+}
+
+.galleryLayoutTitle {
+	text-align: left;
+}
```

Comment:
We're using two different cases here, some selectors use hyphen case and others use camel case.

Reply:
I think this is OK since:
1. camel case CSS selectors are used in Gutenberg (e.g. in the Gallery block [here](https://github.com/WordPress/gutenberg/blob/f2161e246b9fdd9a2a56e7552b0b28050f1a5302/packages/block-library/src/gallery/gallery-styles.native.scss#L3-L5))
2. hyphen case CSS selectors (e.g. `is-style-rectangular` can't be changed since they must match [the block's `className` block attribute](https://github.com/Automattic/jetpack/blob/184b3184226279892310c53cb8e792128ed743f4/projects/plugins/jetpack/extensions/blocks/tiled-gallery/constants.js#L14)

Reply:
@SiobhyB I think I was misguided here when I thought the CSS class names needed fixing. Would you agree this can be marked resolved?

Reply:
@guarani, it could be a good idea to update the class names to match [core's BEM naming methodology](https://developer.wordpress.org/block-editor/contributors/code/coding-guidelines/#css) but I agree that it's not a blocker and could probably be done separately to this PR.